installer.cfg

```diff
@@ -1,6 +1,8 @@
 [Application]
 name=Albion Online Stats
-version=0.11.11
+# order by alphabet.
+publisher=mazurwiktor<wiktormazur1@gmail.com>; mashu<github.com/mashu>; pmx<mingl0280@gmail.com>
+version=0.11.12
```

Comment:
Could you also update version in `src\utils\version.py`? It will be used to notify users about new version availability.

Reply:
Sure. I'll have anoter pull soon.